samples/java-eventsourced-counter-pojo/src/main/java/org/example/domain/ValueReset.java

```diff
@@ -0,0 +1,13 @@
+package org.example.domain;
+
+import com.fasterxml.jackson.annotation.JsonTypeName;
+
+@JsonTypeName("reset")
+public class ValueReset {
+
+  private static final ValueReset instance =  new ValueReset();
```

Comment:
singleton instances are better as `enum` in Java, or is there a problem with json serialization of that?

Reply:
I didn't with enums. I will give it a try.